bingo.js

```diff
@@ -205,13 +218,19 @@ function showOptionsMenu()
 function hideOptionsMenu()
 {
 	$("#bingo-box").removeClass(SHOW_OPTIONS_MENU_CLASS_NAME);
+	closePopup('#urlWithProgress');
 }
 
 function getColourClass(square)
 {
 	return ALL_COLOURS.find(c => square.hasClass(c));
```

Comment:
With the change in function `anotherColour` below, `getColourClass` is no longer used and can be removed.